rootfs/etc/profile.d/Ω_overrides.sh

```diff
@@ -0,0 +1,10 @@
+## Load user's custom overrides
+OVERRIDE_LIST=()
+_search_geodesic_dirs OVERRIDE_LIST overrides
+for file in "${OVERRIDE_LIST[@]}"; do
+	[[ -n $GEODESIC_CUSTOM_TRACE ]] && echo trace: loading override file "$file"
```

Comment:
`GEODESIC_DEBUG_LEVEL` or `GEODESIC_DEBUG` seem more inline to me to flags used by other software. What do you think?

Reply:
The idea I had was to expose one variable that indicate a list of components to trace rather than set a debug level. See [`GEODESIC_TRACE`](https://github.com/cloudposse/geodesic/pull/422/files#diff-70d513de358e16629e7bffecc19a9612R1). `GEODESIC_CUSTOM_TRACE` is an internal variable that turns on tracing for the customization component.

Reply:
Yea, I see - that makes sense. Is trace a keyword you're borrowing from some other language convention? (E.g. Java)

Reply:
I renamed it to `_GEODESIC_TRACE_CUSTOMIZATION`